### DISCUSSION TITLE
Better mesh size control in triangle

### DISCUSSION THREAD

**Discussion by audunarn** [robbievanleeuwen/section-properties#257] [Ideas]
Hi, I find that it is easier to control the mesh size in triangle by changing the max/min angle (in pre.py):

# generate mesh
if coarse:
    mesh = triangle.triangulate(tri, "pAo2")
else:
    mesh = triangle.triangulate(tri, "pq**30**Aao2")

I could add this functionality if you wish.

**Comment by robbievanleeuwen** [comment]
Hi @audunarn,

Thanks for this. Yes I agree this would be an enhancement. Easy enough for me to implement :)

Robbie
